org.mixedrealitytoolkit.input/Experimental/XRI3/HandModel.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clause
+
+using UnityEngine;
+using UnityEngine.XR;
+
+namespace MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// This component stores a reference to the hand model to be used in Editor mode.
```

Comment:
Comment is not correct.   Nothing in this class control if this is visible in Editor or not.

Change to something like
```
/// <summary>
/// This component instantiates a model prefab for hand interactors
/// </summary>
/// <remarks>
/// This does not control the visibility of the instantiated models, the prefab is always created.
/// </remarks>
```

Reply:
Also, as we progress through the XR13 support, you may find this class is not need.   In future changes, let's see if you can just add the "hand model" prefab to the MRTK Rig itself.  This class is not doing anything special, just creating a game object, so I don't see the value. The only useful bit is that it stores some public properties that other components can utilize, and I wonder if this is really needed.

Reply:
Updated comments in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/716/commits/4f971067f6c668d55633ca0e87c398595e5b4d41, resolving.